praetorian_cli/sdk/entities/search.py

```
@@ -422,6 +422,47 @@ def by_query(self, query, pages=100000) -> tuple:
+        else:
+            labels_to_search = [[l] for l in DEFAULT_LABELS]
+
+        all_results = []
+        for labels in labels_to_search:
+            try:
+                node = Node(labels=labels, search=term)
+                query = Query(node=node, limit=limit)
+                results, _ = self.by_query(query)
+                all_results.extend(results)
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)